ecs-init/docker/docker.go

```diff
@@ -109,16 +109,14 @@ const (
 	iptablesUsrLib64Dir = "/usr/lib64"
 
 	hostCapabilitiesResourcesRootDir      = "/var/lib/ecs/deps"
-	containerCapabilitiesResourcesRootDir = "/capabilities"
+	containerCapabilitiesResourcesRootDir = "/managed-agents"
```

Comment:
Don't think we need the 'capability' word in these constants anymore

Reply:
good point, I'm still keeping the `capability` prefix before `exec` since `exec` is one capability, e.g. `capabilityExecName`